plugins/dev-tools/src/common_test_helpers.mocha.js

```diff
@@ -77,7 +82,12 @@ export function runTestSuites(testSuites, createTestCaseCallback) {
     let suiteCall = (testSuite.skip ? suite.skip : suite);
     suiteCall = (testSuite.only ? suite.only : suiteCall);
     suiteCall(testSuite.title, function() {
-      runTestCases(testSuite.testCases, createTestCaseCallback(testSuite));
+      if (testSuite.testSuites && testSuite.testSuites.length) {
+        runTestSuites(testSuite.testSuites, createTestCaseCallback);
+      }
+      if (testSuite.testCases && testSuite.testCases.length) {
+        runTestCases(testSuite.testCases, createTestCaseCallback(testSuite));
+      }
```

Comment:
The check for whether the properties exist is a little confusion given that the record structure indicates them as non-nullable.
I think that this check is reasonable though, given that with the new addition of the testSuites property, I think it would be convenient to not necessarily define both.

So, I'd like the type for `testCases` and `testSuites` to be updated to be nullable (ex: `?Array<T>` instead of `!Array<T>`).